src/System.Security.Claims/src/System/Security/Claims/Claim.cs

```diff
@@ -129,6 +129,7 @@ public Claim(BinaryReader reader, ClaimsIdentity subject)
                 {
                     Properties.Add(reader.ReadString(), reader.ReadString());
                 }
+                numPropertiesRead++;
```

Comment:
No. We are incrementing the `numPropertiesRead` field because we are reading the integer `numProperties` field from the payload.